CMakeLists.txt

```diff
@@ -0,0 +1,68 @@
+cmake_minimum_required(VERSION 3.8)
+cmake_policy(VERSION 3.8)
```

Comment:
This is not necessary, the `cmake_minimum_required` already defaults to the same policy version.